.drone.yml

```diff
@@ -33,6 +33,17 @@ steps:
       - printf "  {\"repo\":\"metrumresearchgroup/bbi\", \"commit\":\"$(cat /tmp/${DRONE_BUILD_NUMBER}/bbi_hash)\"},\n" >> /tmp/${DRONE_BUILD_NUMBER}/commits.json
       - printf "]\n" >> /tmp/${DRONE_BUILD_NUMBER}/commits.json
       - aws s3 cp /tmp/${DRONE_BUILD_NUMBER}/commits.json s3://mrg-validation/bbi/${DRONE_BUILD_NUMBER}/commits.json
+  - name: Copy summary info for mrgvalidate to s3
+    commands:
+      - echo $SUMMARYJSON | base64 --decode > /tmp/${DRONE_BUILD_NUMBER}/summary.json
+      - datetime=$(date +"%D %T")
+      - sed -i "s@<DATETIME>@$datetime@g" /tmp/${DRONE_BUILD_NUMBER}/summary.json
+      - commit=$(git rev-parse HEAD)
+      - sed -i "s/<COMMIT>/$commit/g" /tmp/${DRONE_BUILD_NUMBER}/summary.json
+      - sed -i "s/<DRONE_BUILD_NUMBER>/$DRONE_BUILD_NUMBER/g" /tmp/${DRONE_BUILD_NUMBER}/summary.json
+      - aws s3 cp /tmp/${DRONE_BUILD_NUMBER}/summary.json s3://mrg-validation/bbi/${DRONE_BUILD_NUMBER}/summary.json
+    environment:
+      SUMMARYJSON: ewogICJkYXRlIjogIjxEQVRFVElNRT4iLAogICJleGVjdXRvciI6ICJEcm9uZSBDSSIsCiAgImluZm8iOiB7CiAgICAiY29tbWl0IjogIjxDT01NSVQ+IiwKICAgICJkcm9uZV9idWlsZF9udW1iZXIiOiI8RFJPTkVfQlVJTERfTlVNQkVSPiIKICB9Cn0=
```

Comment:
The encoding seems like an unnecessary layer to me that makes things harder to follow.  I'd find this clearer:

```
printf '{\n  "date": "<DATETIME>",\n  "executor": "Drone CI",\n  "info": {\n    "commit": "<COMMIT>",\n    "drone_build_number":"<DRONE_BUILD_NUMBER>"\n  }\n}'
```

And I think a heredoc would be clearest:

```sh
cat >/tmp/${DRONE_BUILD_NUMBER}/summary.json <<EOF
{
  "date": "$(date +"%D %T")",
  "executor": "Drone CI",
  "info": {
    "commit": "$(git rev-parse HEAD)",
    "drone_build_number": "$DRONE_BUILD_NUMBER"
  }
}
EOF
```

Perhaps the drone configuration format doesn't allow a heredoc, but I know they work fine with some other CI systems.

That being said, the above works (and I know it's similar to what is done at least one other mrg repo), so no real objections from me.

Reply:
@kyleam I like that change and it works perfectly:

```bash
$ aws s3 cp s3://mrg-validation/bbi/784/summary.json -
{
  "date": "10/13/21 09:28:16",
  "executor": "Drone CI",
  "info": {
    "commit": "4826c8f5f5436e9503d41df19b8008605ca8eda7",
    "drone_build_number": "784"
  }
}
```